bentoml/_internal/io_descriptors/json.py

```diff
@@ -197,13 +197,17 @@ async def to_http_response(
         if isinstance(obj, pydantic.BaseModel):
             obj = obj.dict()
 
-        json_str = json.dumps(
-            obj,
-            cls=self._json_encoder,
-            ensure_ascii=False,
-            allow_nan=False,
-            indent=None,
-            separators=(",", ":"),
+        json_str = (
+            json.dumps(
+                obj,
+                cls=self._json_encoder,
+                ensure_ascii=False,
+                allow_nan=False,
+                indent=None,
+                separators=(",", ":"),
+            )
+            if obj is not None
```

Comment:
```suggestion
            if obj
```
There could be situation when obj is an empty string, empty list, tuple

Reply:
Empty collections are probably OK serializing them the way it is right now. However, `None` is currently not an accepted type of `to_http_response`. I will do another PR on top of this one to add None as an accepted type.